stl/inc/type_traits

```diff
@@ -2258,7 +2264,9 @@ namespace _DEPRECATE_TR1_NAMESPACE tr1 {
     using _STD is_member_object_pointer;
     using _STD is_member_pointer;
     using _STD is_object;
+#if !_HAS_DEPRECATED_TYPETRAITS

```

Comment:
Another `is_pod` occurrence that shouldn't be guarded.

Reply:
true